packages/service/ni_measurement_plugin_sdk_service/measurement/service.py

```diff
@@ -210,7 +210,10 @@ def __init__(
             raise RuntimeError(f"File does not exist. {service_config_path}")
 
         with open(service_config_path) as service_config_file:
-            service_config = json.load(service_config_file)
+            try:
+                service_config = json.loads(service_config_file.read())
+            except json.decoder.JSONDecodeError:
+                raise NameError(f"Invalid character(s) in {service_config_path}")
```

Comment:
This is an overgeneralization. There are lots of other reasons to raise JSONDecodeError.